src/modules/scores.js

```diff
@@ -1,6 +1,7 @@
 export default class Scores {
-  constructor(name, score) {
-    this.name = name;
-    this.score = score;
+  constructor(userName, points) {
+    userName = this.userName;
+    // eslint-disable-next-line no-unused-vars
+    points = this.points;
```

Comment:
- [OPTIONAL] Based on the suggestion by the previous reviewer, He advised you to not disable the linter checks for it to serve its purpose. This suggestion has not been implemented. Kindly adhere to the suggestion.